helm-chart/values.yaml

```diff
@@ -23,6 +23,11 @@ rbac:
   # create specifies whether to install and use RBAC rules.
   create: true
 
+psp:
+  # create specifies whether to install and use Pod Security Policies.
+  # This only apples to L2 mode.
```

Comment:
Can you remove this line? It'll just confuse the issue, most people don't know enough about the internals of MetalLB to make sense of this, it'll just create more support burden.

Reply:
No worries, removed this and also got rid of my changes to the Makefile.